upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -877,16 +877,6 @@ func setupControlPlane(opt *NewClusterOptions, cluster *api.Cluster, zoneToSubne
 				g.Spec.Zones = []string{zone}
 			}
 
-			if cloudProvider == api.CloudProviderAWS {
-				g.Spec.InstanceMetadata = &api.InstanceMetadataOptions{
-					HTTPPutResponseHopLimit: fi.PtrTo(int64(3)),
-					HTTPTokens:              fi.PtrTo("required"),
-				}
-			}
-			if cluster.IsKubernetesGTE("1.26") && fi.ValueOf(cluster.Spec.IAM.UseServiceAccountExternalPermissions) {
-				g.Spec.InstanceMetadata.HTTPPutResponseHopLimit = fi.PtrTo(int64(1))
```

Comment:
I think we need to leave this in for k8s == 1.26.x && fi.ValueOf(cluster.Spec.IAM.UseServiceAccountExternalPermissions)

Reply:
See the comment ^^^.

Reply:
Which comment were you referring to?

Reply:
This logic is replaced by the one where you asked:
> Is there any reason we can't set this to 1 for k8s >= 1.27? Possibly in a follow-up PR to be done in 1.27.

Reply:
I asked to change the default for no-supplied-value. I didn't ask to change the default for new clusters.

Reply:
The "this to 1" applies to control-plane nodes in non-IRSA clusters.

Reply:
It is an inverted condition. As the default is "1" for ALL, the condition to switch to 3 for control plane without IRSA had to be updated.